### ISSUE TITLE
Installation for CUDA 11 issues

### ISSUE THREAD

**Issue by AlbertoRemus** [aimagelab/mcmr#6]
Hi, I need to work with CUDA 11, so I investigated on how to do that

(PyTorch3D seemed to be really picky about the Python/PyTorch version):

Here's the sequence of commands before running this repo's scripts

```
conda create -n mcmr python=3.8
conda activate mcmr
conda install -c pytorch pytorch=1.10.0 torchvision cudatoolkit=11.3 -c conda-forge
conda install -c fvcore -c iopath -c conda-forge fvcore iopath
conda install -c bottler nvidiacub
conda install -c pytorch3d pytorch3d
python -m pip install opencv-python
python -m pip install imageio
python -m pip install lpips
python -m pip install tensorboard
python -m pip install meshzoo
python -m pip install scikit-image
python setup_softras.py build develop
```

Tested on Ubuntu20.04, with CUDA 11.4 on a docker

I place the sequence of commands I used in case others find it useful

**Comment by AlbertoRemus** [comment]
Just one remark about `tensorboard`, since on this setup https://github.com/aimagelab/mcmr/issues/6#issue-1224079249, when testing https://github.com/aimagelab/mcmr#22-cub I get:

```
Traceback (most recent call last):
  File "/home/robotics/object_pose_detection/mcmr/main.py", line 16, in <module>
    from torch.utils.tensorboard import SummaryWriter
  File "/home/robotics/conda/envs/mcmr/lib/python3.8/site-packages/torch/utils/tensorboard/__init__.py", line 4, in <module>
    LooseVersion = distutils.version.LooseVersion
AttributeError: module 'distutils' has no attribute 'version
```

since for the moment I don't need it I just commented the offending line `from torch.utils.tensorboard import SummaryWriter`

**Comment by AlbertoRemus** [comment]
Related PR https://github.com/pytorch/pytorch/pull/69904

**Comment by AlbertoRemus** [comment]
> Just one remark about tensorboard, since on this setup https://github.com/aimagelab/mcmr/issues/6#issue-1224079249, when testing https://github.com/aimagelab/mcmr#22-cub I get:

> Traceback (most recent call last):
  File "/home/robotics/object_pose_detection/mcmr/main.py", line 16, in <module>
    from torch.utils.tensorboard import SummaryWriter
  File "/home/robotics/conda/envs/mcmr/lib/python3.8/site-packages/torch/utils/tensorboard/__init__.py", line 4, in <module>
    LooseVersion = distutils.version.LooseVersion
AttributeError: module 'distutils' has no attribute 'version
since for the moment I don't need it I just commented the offending line from torch.utils.tensorboard import SummaryWriter

`python -m pip install setuptools==59.5.0`

solved for the provided configuration

**Comment by stefanopini** [comment]
Hi @AlbertoRemus , thanks for reporting the issue and finding solution for it!
@alexj94 btw, can we remove the dependency from tensorboard and simply use `torch.utils.tensorboard` instead?

**Comment by alexj94** [comment]
> @alexj94 btw, can we remove the dependency from tensorboard and simply use `torch.utils.tensorboard` instead?

Unfortunately, it seems to be an issue related to the PyTorch version and it should be fixed in PyTorch 1.11 (according to the related PR https://github.com/pytorch/pytorch/pull/69904). If you use a previous version that generates the error, the only fix is the one found by @AlbertoRemus:

> python -m pip install setuptools==59.5.0
